packages/manager/apps/pci-rancher/src/routes.tsx

```diff
@@ -16,24 +16,38 @@ const lazyRouteConfig = (importFn: CallableFunction) => {
 
 export const COMMON_PATH = '/pci/projects';
 
+export enum Route {
+  Listing = 'listing',
+  Dashboard = 'dashboard',
+  Onboarding = 'onboarding',
+  New = 'New',
+}
+
+export const getRequestTaggerPciPath = (project: string, route?: Route) =>
+  `pci.projects.project.${project}${route ? `.${route}` : ''}`;
+
 export default [
   {
     path: '/pci/projects/:projectId',
     ...lazyRouteConfig(() => import('@/pages/layout')),
     children: [
       {
+        id: getRequestTaggerPciPath('rancher'),
         path: 'rancher',
         ...lazyRouteConfig(() => import('@/pages/')),
       },
       {
+        id: getRequestTaggerPciPath('rancher', Route.Onboarding),
         path: 'rancher/onboarding',
         ...lazyRouteConfig(() => import('@/pages/onboarding')),
       },
       {
+        id: getRequestTaggerPciPath('rancher', Route.New),
         path: 'rancher/new',
         ...lazyRouteConfig(() => import('@/pages/create')),
       },
       {
+        id: getRequestTaggerPciPath('rancher', Route.Dashboard),
```

Comment:
**nitpick:** It has been decided to put the tracking related info in the `handle` config of the route definition. Can you please change it accordingly?